runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11952,7 +11952,7 @@ TR::Register*
 J9::Z::TreeEvaluator::VMinlineCompareAndSwap(TR::Node *node, TR::CodeGenerator *cg, TR::InstOpCode::Mnemonic casOp, bool isObj, bool isExchange)
    {
    TR::Register *scratchReg = NULL;
-   TR::Register *objReg, *oldVReg, *newVReg;
+   TR::Register *objReg, *oldVReg, *newVReg, *decompressedValueRegister;
```

Comment:
I understand this exists, but we should initialize the registers here with `NULL`.

Reply:
I can change this.